plugins/steelseries/fu-steelseries-fizz.c

```diff
@@ -0,0 +1,720 @@
+/*
+ * Copyright (C) 2022 Gaël PORTAY <gael.portay@collabora.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-steelseries-common.h"
+#include "fu-steelseries-fizz.h"
+
+#define STEELSERIES_TRANSACTION_TIMEOUT	 5000
+#define STEELSERIES_BUFFER_CONTROL_SIZE	 64
+#define STEELSERIES_BUFFER_TRANSFER_SIZE 52
+
+#define STEELSERIES_FIZZ_ACCESS_FILE_COMMAND_OFFSET    0x00U
+#define STEELSERIES_FIZZ_ACCESS_FILE_FILESYSTEM_OFFSET 0x01U
+#define STEELSERIES_FIZZ_ACCESS_ID_OFFSET	       0x02U
+#define STEELSERIES_FIZZ_ACCESS_SIZE_OFFSET	       0x03U
+#define STEELSERIES_FIZZ_ACCESS_OFFSET_OFFSET	       0x05U
+#define STEELSERIES_FIZZ_ACCESS_FILE_DATA_OFFSET       0x09U
+
+struct _FuSteelseriesFizz {
+	FuUsbDevice parent_instance;
+	FuSteelseriesDeviceKind device_kind;
+	guint8 iface_idx;
+	guint8 ep;
+	gsize in_size;
+};
+
+typedef enum {
+	STEELSERIES_FIZZ_COMMAND_ERROR_SUCCESS = 0,
+	STEELSERIES_FIZZ_COMMAND_ERROR_FILE_NOT_FOUND,
+	STEELSERIES_FIZZ_COMMAND_ERROR_FILE_TOO_SHORT,	  /* targeted offset is past the file end */
+	STEELSERIES_FIZZ_COMMAND_ERROR_FLASH_FAILED,	  /* when internal flash returns error */
+	STEELSERIES_FIZZ_COMMAND_ERROR_PERMISSION_DENIED, /* USB API doesn't have permission to
+							     access this file */
+	STEELSERIES_FIZZ_COMMAND_ERROR_OPERATION_NO_SUPPORTED, /* USB API doesn't have permission to
+								  access this file */
+	/*< private >*/
+	STEELSERIES_FIZZ_COMMAND_ERROR_LAST
+} SteelseriesFizzCommandError;
+
+static gint STEELSERIES_FIZZ_COMMAND_ERRORS[] = {
+    0,
```

Comment:
0?

Reply:
that is a success, what is the appropriate value in the world of glib?